pkg/spire/spire_test.go

```diff
@@ -455,34 +455,27 @@ func TestSpire_TaskRunResultsSignTamper(t *testing.T) {
 	defer cc.Close()
 	defer ec.Close()
 
-	genPr := func() []v1beta1.PipelineResourceResult {
-		return []v1beta1.PipelineResourceResult{
-			{
-				Key:          "foo",
-				Value:        "foo-value",
-				ResourceName: "source-image",
-				ResultType:   v1beta1.TaskRunResultType,
-			},
-			{
-				Key:          "bar",
-				Value:        "bar-value",
-				ResourceName: "source-image2",
-				ResultType:   v1beta1.TaskRunResultType,
-			},
-		}
-	}
+	pr := []v1beta1.PipelineResourceResult{{
+		Key:          "foo",
+		Value:        "foo-value",
+		ResourceName: "source-image",
+		ResultType:   v1beta1.TaskRunResultType,
+	}, {
+		Key:          "bar",
+		Value:        "bar-value",
+		ResourceName: "source-image2",
+		ResultType:   v1beta1.TaskRunResultType,
+	}}
```

Comment:
I notice that the preexisting tests consistently generate new copies of test data for each test. Is that out of concern that the test might accidentally modify its inputs? If there's a real concern that the code *under* test might do so, I think it would be better to get a chance of exposing it by making the tests share test data, but even better would be detecting the changes on purpose.

In any case, I think it's important to be consistent here. If you're going to change this, you'll also want to change `testTaskRuns()` and `testPipelineResourceResults()`.

Reply:
Well doh! This is obvious now; fixed.